### PR TITLE
created fullstack development folder

### PR DESCRIPTION
**Pull request by Rachel-Cellulose**
**Add Fullstack Development Resources Folder**

Description: This pull request adds a new folder named fullstack-development, intended to serve as a dedicated space for resources aimed at helping learners navigate the full-stack development landscape.

Key Changes:
Created a new directory: fullstack-development.

Impact:
This contribution lays the foundation for future enhancements by providing a structured location for resources related to full-stack development. As resources are added, this folder will become a valuable tool for individuals seeking structured materials to enhance their full-stack development skills.

### CODE REVIEW

**Review by Charity10 — CHANGES_REQUESTED** [review]

File: FullstackDevelopment/Fullstack.md
[the remark this answers is not in the document]
Comment (Rachel-Cellulose) [reply FullstackDevelopment/Fullstack.md:1]: Thank you. Changes have been updated.

File: FullstackDevelopment/Fullstack.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+## 🛣️ Fullstack Development Roadmap
+
+Here’s a visual roadmap to guide you through the fullstack development journey:
+
+![Fullstack Development Roadmap](https://github.com/Rachel-Cellulose/LearningResources/blob/fullstack-development/FullstackDevelopment/assets/fullstack-roadmap.jpg) 
+
+
+## 📚 Table of Contents
+1. [🚀 Getting Started with Fullstack Development](#-getting-started-with-fullstack-development)
```
Comment (Charity10) [inline FullstackDevelopment/Fullstack.md:14]: @Rachel-Cellulose 
Your project is good

The only problem now is if I click on the table of contents it doesn't take me to that section instead I'm directed to a new page entirely

And this page looks like a web page with just html and no CSS

You can look at the readme file to see the difference

File: FullstackDevelopment/Fullstack.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+### Key Concepts:
+- **Microservices**: Break down monolithic applications into small, scalable services.
+- **GraphQL**: Learn how to use GraphQL as an alternative to REST APIs for fetching data.
+- **Serverless Architecture**: Explore how to build and deploy serverless applications using AWS Lambda or Azure Functions.
+
+---
+
+## 🤝 Contribute
+
+We welcome contributions to help improve these resources! If you have more fullstack development resources, tutorials, or suggestions, please follow our [contribution guidelines](#) to make your additions.
```
Comment (Charity10) [inline FullstackDevelopment/Fullstack.md:394]: Link the contribution.md file here

Pull first to get updated changes from the repo

Comment (Charity10) [reply FullstackDevelopment/Fullstack.md:394]: > Link the contribution.md file here
> 
> Pull first to get updated changes from the repo


@Rachel-Cellulose

**Pull request merged**